packages/@aws-cdk/aws-servicecatalog/lib/portfolio.ts

```diff
@@ -120,7 +120,30 @@ export interface IPortfolio extends cdk.IResource {
   setLaunchRole(product: IProduct, launchRole: iam.IRole, options?: CommonConstraintOptions): void;
 
   /**
-   * Configure deployment options using AWS Cloudformaiton StackSets
+   * Force users to assume a certain role when launching a product.
+   * The role will be referenced by name in the local account.
+   * A role with this name will automatically be created and assumable by Service Catalog in this account.
+   * This is useful when sharing the portfolio with multiple accounts.
+   *
+   * @param product A service catalog product.
+   * @param launchRoleName The name of the IAM role a user must assume when provisioning the product. A role with this name must exist in the account where the portolio is created and the accounts it is shared with.
+   * @param options options for the constraint.
+   */
+  setLocalLaunchRoleName(product: IProduct, launchRoleName: string, options?: CommonConstraintOptions): iam.IRole;
+
+  /**
+   * Force users to assume a certain role when launching a product.
+   * The role name will be referenced by name in the local account and must be set explicitly.
+   * This is useful when sharing the portfolio with multiple accounts.
+   *
+   * @param product A service catalog product.
+   * @param launchRole The IAM role a user must assume when provisioning the product. A role with this name must exist in the account where the portolio is created and the accounts it is shared with. The role name must be set explicitly.
+   * @param options options for the constraint.
+   */
+  setLocalLaunchRole(product: IProduct, launchRole: iam.IRole, options?: CommonConstraintOptions): void;
```

Comment:
I have to say, I don't really understand the difference between this method, and `setLaunchRole()`. Can you make sure to make it clear in the documentation of this class?

Reply:
tried to make it clearer in both methods, let me know if that makes sense